xds/xds.go

```diff
@@ -74,22 +77,69 @@ func init() {
 		v3statusgrpc.RegisterClientStatusDiscoveryServiceServer(grpcServer, csdss)
 		return csdss.Close, nil
 	})
+
+	internal.NewXDSResolverWithConfigForTesting = func(bootstrapConfig []byte) (resolver.Builder, error) {
```

Comment:
Move this to the xdsresolver package?  And can it be just:

```go
internal.NewXDSResolverWithConfigForTesting = xdsresolver.NewBuilderForTesting
```

Reply:
Done.